### PR TITLE
company list only its own job vacancies

### PR DESCRIPTION
**Pull request by MarceloBorgesP**
- companies now can see only its own vacancies in its panel
- Templates are now imported in html when possible

### CODE REVIEW

File: client/html/layouts/panel.html [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
     </header>
 
     <div id="base">
-      {{> Template.dynamic template=menubar}}
+      {{> menubar }}
       <div id="content">
         {{> Template.dynamic template=main}}
       </div>
-      {{> Template.dynamic template=offcanvas}}
+      {{> offcanvas }}
```
Comment (caike08) [inline client/html/layouts/panel.html:12]: Why did you change these fields? Can you explain me more?

File: client/router.js [hunk 2]
```diff
@@ -25,23 +25,23 @@ FlowRouter.route('/', {
...
+    BlazeLayout.render('HomeLayout', {main: "studentSignup" });
   }
 });
 
 FlowRouter.route('/cadastro/empresa', {
   name: 'companySignup',
   title: "Banco de Talentos - Cadastrar Empresa",
   action: function(params) {
-    BlazeLayout.render('HomeLayout', { header: "header", main: "companySignup" });
+    BlazeLayout.render('HomeLayout', {main: "companySignup" });
```
Comment (caike08) [inline client/router.js:44]: Why aren't you calling "header" now? Didn't get it well

File: client/html/layouts/panel.html (hunk 1 above)
Comment (MarceloBorgesP) [reply client/html/layouts/panel.html:12]: When we do:

``` html
{{> Template.dynamic template=offcanvas}}
```

We are naming an area offcanvas, and we can put any template inside of this area.

But when we do:

``` html
{{> offcanvas }}
```

We are telling to render the template inside on the specified area, no other template can be there besides 'offcanvas'

As there is no reason to use another template in that space besides offcanvas, we changed to this new way, and now we will always render 'ofcanvas' in there.

File: client/router.js (hunk 2 above)
Comment (MarceloBorgesP) [reply client/router.js:44]: As explained in the other comment, we changed the way we render, now we render the template 'header' like this:

``` html
{{> header }}
```

Then we don't need to tell Blaze which template he should render, he already knows it
